### PR TITLE
LPS-160824 Uses clay:dropdown-actions for folder actions

### PR DESCRIPTION
**Pull request by ambrinchaudhary**
Note: move action does not work in Media Gallery portlet, but this is a bug in current master.

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Comment by ambrinchaudhary** [comment]
Hey @ealonso, as I mentioned [here](https://github.com/liferay-lima/liferay-portal/pull/3371#discussion_r948142886), could you please confirm if the third [commit](https://github.com/liferay-lima/liferay-portal/pull/3381/commits/64002a5ad17a2e21e825963e322d7b74413a522d) is correct?

**Comment by ealonso** [comment]
@ambrinchaudhary Thanks you very much for the fixes, the third commit is correct, it was my mistake.

**Comment by ambrinchaudhary** [comment]
ci:test:sf

**Comment by ambrinchaudhary** [comment]
ci:test:relevant

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/FolderActionDisplayContext.java [hunk 1]
```diff
@@ -151,6 +151,9 @@ public List<DropdownItem> getActionDropdownItems() throws PortalException {
 						isAddMediaActionVisible() &&
 						isMultipleUploadSupported(),
 					dropdownItem -> {
+						dropdownItem.put(
```
Comment (ealonso) [inline modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/FolderActionDisplayContext.java:154]: @ambrinchaudhary that's was really nice, 😊

**Comment by AliciaGarciaGarcia** [comment]
ci:reevaluate:1387401_3080

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/FolderActionDisplayContext.java [hunk 2]
```diff
@@ -75,6 +85,168 @@ public FolderActionDisplayContext(
...
+							"webDavURL",
+							DLURLHelperUtil.getWebDavURL(
+								themeDisplay, _getFolder(), null));
+
+						dropdownItem.setLabel(
+							LanguageUtil.get(
+								_httpServletRequest, "access-from-desktop"));
+					}
+				).add(
+					() -> isPermissionsActionVisible(),
```
Comment (AliciaGarciaGarcia) [inline modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/FolderActionDisplayContext.java:210]: maybe is worth to make all the `isPermissionsActionVisible` ... `isDeleteFolderActionVisible` private now that they are not called from the jsp?

**Comment by AliciaGarciaGarcia** [comment]
maybe the poshi test should be changed?
because seems relevant
[LocalFile.DMFolder#CanEditFolder](https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/177526//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_DMFolder_CanEditFolder_) - [Poshi Report](https://testray.liferay.com/reports/production/logs/2022-08/test-1-38/test-portal-acceptance-pullrequest(master)/3080/functional-tomcat90-mysql57-jdk8/2/0/LocalFile.DMFolder_CanEditFolder/index.html.gz) - [Poshi Summary](https://testray.liferay.com/reports/production/logs/2022-08/test-1-38/test-portal-acceptance-pullrequest(master)/3080/functional-tomcat90-mysql57-jdk8/2/0/LocalFile.DMFolder_CanEditFolder/summary.html.gz) - [Console Output](https://testray.liferay.com/reports/production/logs/2022-08/test-1-38/test-portal-acceptance-pullrequest(master)/3080/functional-tomcat90-mysql57-jdk8/2/0/jenkins-console.txt.gz)
[LocalFile.DMFolder#CanEditSubFolder](https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/177526//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_DMFolder_CanEditSubFolder_) - [Poshi Report](https://testray.liferay.com/reports/production/logs/2022-08/test-1-38/test-portal-acceptance-pullrequest(master)/3080/functional-tomcat90-mysql57-jdk8/2/0/LocalFile.DMFolder_CanEditSubFolder/index.html.gz) - [Poshi Summary](https://testray.liferay.com/reports/production/logs/2022-08/test-1-38/test-portal-acceptance-pullrequest(master)/3080/functional-tomcat90-mysql57-jdk8/2/0/LocalFile.DMFolder_CanEditSubFolder/summary.html.gz) - [Console Output](https://testray.liferay.com/reports/production/logs/2022-08/test-1-38/test-portal-acceptance-pullrequest(master)/3080/functional-tomcat90-mysql57-jdk8/2/0/jenkins-console.txt.gz)
[LocalFile.DMFolder#CanMoveFolderToFolder](https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/177526//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_DMFolder_CanMoveFolderToFolder_) - [Poshi Report](https://testray.liferay.com/reports/production/logs/2022-08/test-1-38/test-portal-acceptance-pullrequest(master)/3080/functional-tomcat90-mysql57-jdk8/2/0/LocalFile.DMFolder_CanMoveFolderToFolder/index.html.gz) - [Poshi Summary](https://testray.liferay.com/reports/production/logs/2022-08/test-1-38/test-portal-acceptance-pullrequest(master)/3080/functional-tomcat90-mysql57-jdk8/2/0/LocalFile.DMFolder_CanMoveFolderToFolder/summary.html.gz) - [Console Output](https://testray.liferay.com/reports/production/logs/2022-08/test-1-38/test-portal-acceptance-pullrequest(master)/3080/functional-tomcat90-mysql57-jdk8/2/0/jenkins-console.txt.gz)

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/FolderActionDisplayContext.java (hunk 2 above)
Comment (ealonso) [reply modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/FolderActionDisplayContext.java:210]: It is a good point, I can do it, if you want

**Comment by ealonso** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by ealonso** [comment]
Hey @ambrinchaudhary @AliciaGarciaGarcia,

I've resend the pr here with the Alicia's proposal https://github.com/liferay-lima/liferay-portal/pull/3388.

Please let me know if you need something else.

**Comment by austinchiang** [comment]
Closing in favor of https://github.com/liferay-lima/liferay-portal/pull/3388

**Pull request closed without merging**